File: src/Other/Internals/Menu/PageCategoryInfo.fs
namespace OpenSilver.Samples.Showcase

open System.Collections.ObjectModel
open System.Windows.Media

type PageCategoryInfo() =
    interface IMenuElement
    member val Name = "" with get, set
    member val Foreground : Brush = null with get, set
    member val Pages = new ObservableCollection<PageInfo>() with get, set

File: src/Other/Internals/Menu/PageCategoryInfo.vb
Imports System.Collections.ObjectModel
Imports System.Windows.Media

Namespace OpenSilver.Samples.Showcase
    Public Class PageCategoryInfo
        Implements IMenuElement

        Public Property Name As String
        Public Property Foreground As Brush
        Public Property Pages As New ObservableCollection(Of PageInfo)
    End Class
End Namespace
